extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/CodeAuthenticationMechanism.java

```diff
@@ -188,31 +188,19 @@ private boolean shouldAutoRedirect(TenantConfigContext configContext, RoutingCon
                     configContext.auth.authorizeURL(params));
         } else {
             // If the client (usually an SPA) wants to handle the redirect manually, then
-            // return status code 399 and in the www-authenticate header put the redirect url
+            // return status code 499 and in the www-authenticate header put the redirect url
             // www-authenticate header content; No standard content matched the exact use case
             // so the OAuth redirect-url= is an arbitrary string that can be tested
             // for on the client side
-            String headerContent = "OAuth redirect-url=" + configContext.auth.authorizeURL(params);
+            String headerContent = "OIDC redirect-url=" + configContext.auth.authorizeURL(params);
             // Vertx HTTPHeaders does noy have a "WWW_AUTHENTICATE" static
             // So, header name is hard coded for the moment
-            challenge = new ChallengeData(399, "WWW-Authenticate", headerContent);
+            challenge = new ChallengeData(499, "WWW-Authenticate", headerContent);
```

Comment:
399 doesn't work - at least on Chrome. If Chrome receives any 3XX code on an XHR request it automatically redirects. 499 allows the client to handle the redirect.

Reply:
@boosey Sounds good